index.html

```diff
@@ -0,0 +1,785 @@
+            <div>
+               <h4><a class="section2_links" href="">3 reasons why almost every VC investor passed on Zoom</a></h4>
+                <div>
+                    <a class="special_text" href="#">Philip Levinson</a>
+                    <p class="special_text">· 5 hours ago</p>
+                </div>
+            </div>
+        </div>
+        <div class="section2_items">
+            <img alt="image"
+                src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/biggst-ev-motor-electric-car-future-diesel-factory-production-future-PSA-world-480x270.png">
+            <div>
+                <h4><a class="section2_links" href="">It’s happening! World’s biggest diesel factory pivots to electric
+                    motors</a></h4>
+                <div>
+                    <a class="special_text" href="#">Matthew Beedham</a>
+                    <p class="special_text">· 6 hours ago</p>
+                </div>
+            </div>
+        </div>
+        <div class="section2_items">
+            <img alt="image"
+                src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/1-copy-51-516x270.jpg">
+            <div>
+                <h4><a class="section2_links" href="">This nearly inexplicable exoplanet has the density of cotton candy</a></h4>
+                <div>
+                    <a class="special_text" href="#">The Cosmic Companion</a>
+                    <p class="special_text">· 6 hours ago</p>
+                </div>
+            </div>
+        </div>
+        <div class="section2_items">
+            <img alt="image"
+                src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/biden-electric-cars-future-us-state-vehicles-car-EV-480x270.jpg">
+            <div>
+                <h4><a class="section2_links" href="">The future of US government vehicles is electric</a></h4>
+                <div>
+                    <a class="special_text" href="#">Matthew Beedham</a>
+                    <p class="special_text">· 7 hours ago</p>
+                </div>
+            </div>
+        </div>
+        <div class="section2_items">
+          <img alt="image"
+              src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/Signal-hed-513x270.jpg">
+          <div>
+              <h4><a class="section2_links" href="">Signal is drama-free for now, but it should prepare for the worst</a></h4>
+              <div>
+                  <a class="special_text" href="#">Abhimanyu Ghoshal</a>
+                  <p class="special_text">· 8 hours ago</p>
+              </div>
+          </div>
+
+      </div>
+    </section>-->
+
+    <section class="ltst-fndg">
```

Comment:
## Improve the layout of the latest funding section.
### on a large screen.
- [ ] Kindly observe that the three main articles of the page seem to be occupying the entire width of the screen in large screen breakpoints. The rest sections' contents saved for the footer of the page seem to be covering lesser width fo the page and are all centred.  see the comparison below 👇🏾 

|[Source Page](https://perma.cc/M5ZV-Q2D6)|[Clone Page](https://raw.githack.com/verissimohenry/TNW-Prema/mileston1-mainsection/index.html)|
|:--:|:--:|
|![latest-funding](https://user-images.githubusercontent.com/22345961/110199910-e005a980-7e6b-11eb-9131-5e13f38c04b0.png)|![latest-funding-clone](https://user-images.githubusercontent.com/22345961/110199912-e3993080-7e6b-11eb-9bf1-f808d0d87c99.png)|

### On small screens 
- [ ] Ensure that all the elements in this section are all aligned in a column and are all covering the entire view width of the page. 

|[Source Page](https://perma.cc/M5ZV-Q2D6)|[Clone Page](https://raw.githack.com/verissimohenry/TNW-Prema/mileston1-mainsection/index.html)|
|:--:|:--:|
|![latest-funding-small-screen](https://user-images.githubusercontent.com/22345961/110199935-0a576700-7e6c-11eb-8da3-4534df2b53e1.png)|![latest-funding-small-screen-clone](https://user-images.githubusercontent.com/22345961/110199939-0deaee00-7e6c-11eb-8861-c19b3841460c.png)|